test/daemon/daemon.go

```diff
@@ -148,7 +152,17 @@ func (d *Config) StartDaemon() error {
 	if util.WaitTimeout(time.Duration(d.timeout)*time.Second, d.IsDaemonUp) == false {
 		if d.Debug == true {
 			d.DumpLog()
+
+			fmt.Printf("\n")
 			fmt.Printf("Failed to launch pouchd:%v\n", d.Args)
```

Comment:
maybe we could use `fmt.Printf("\nFailed to launch pouchd:%v\n", d.Args)`

Reply:
good